clients/client-python/setup.py

```diff
@@ -24,16 +24,16 @@
     with open("README.md") as f:
         long_description = f.read()
 except FileNotFoundError:
-    long_description = "Gravitino Python client"
+    long_description = "Apache Gravitino Python client"
 
 setup(
-    name="gravitino",
-    description="Python lib/client for Gravitino",
+    name="apache-gravitino",
+    description="Python lib/client for Apache Gravitino",
     version="0.7.0.dev",
```

Comment:
@xunliu can you please change the version here to "0.7.0.dev0", I found that python will automatically add a "0" after "dev" if not specified.